tests/test_browser_features.py

```
@@ -134,45 +124,28 @@ def _enter_gameplay_and_gameover(page):
+    # Click Credits button (virtual 500, 540)
+    pt = _virtual_to_canvas_point(page, 500, 540)
     page.mouse.click(pt["x"], pt["y"])
-    changed, menu_open = _wait_for_canvas_change(page, before, wait_ms=2_000)
-    if not changed:
-        pytest.skip("Canvas did not change after clicking MENU on this backend/runtime")
-
-    # Click About button (virtual 400, 395)
-    pt = _virtual_to_canvas_point(page, 400, 395)
-    page.mouse.click(pt["x"], pt["y"])
-    changed, after = _wait_for_canvas_change(page, menu_open, wait_ms=2_500)
+    changed, after = _wait_for_canvas_change(page, before, wait_ms=2_500)
```

Ruff (0.15.12)

[warning] Unpacked variable `after` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)